src/modm/board/nucleo_f767zi/board.hpp

```diff
@@ -0,0 +1,150 @@
+/*
+ * Cpoyright (c) 2021, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_STM32_NUCLEO_F767ZI_HPP
+#define MODM_STM32_NUCLEO_F767ZI_HPP
+
+#include <modm/platform.hpp>
+#include <modm/architecture/interface/clock.hpp>
+#include <modm/debug/logger.hpp>
+/// @ingroup modm_board_nucleo_F767ZI
+#define MODM_BOARD_HAS_LOGGER
+
+using namespace modm::platform;
+
+/// @ingroup modm_board_nucleo_F767ZI
```

Comment:
The `:docs` generator expects this to be lower case, not sure if Doxypress is case-sensitive and would link this correctly.